resource_customizations/work.karmada.io/ResourceBinding/health.lua

```diff
@@ -0,0 +1,38 @@
+health_status = {}
+if ( obj.status ~= nil and obj.spec ~= nil ) then
```

Comment:
Can you remind me why we check `obj.spec` against `nil` here? Seems it will never be nil.

Reply:
> Can you remind me why we check obj.spec against nil here?

double check. 😊

Reply:
I think the redundant check should be removed.